packages/patternfly-4/react-core/CONTRIBUTING.md

```diff
@@ -103,6 +103,6 @@ $ git push origin my-branch
 
```

Comment:
"Pull request" can be lowercase and "Generate your Component" can be "Generate your component."